pkgs/development/python-modules/quart/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitLab
+, pythonOlder
+, click
+, aiofiles
+, werkzeug
+, jinja2
+, hypercorn
+, itsdangerous
+, blinker
+, hypothesis
+, pytestCheckHook
+, pytest-asyncio
+, poetry-core
+}:
+
+buildPythonPackage rec {
+  pname = "quart";
+  version = "0.17.0";
+  disabled = pythonOlder "3.7";
+  format = "pyproject";
+
+  src = fetchFromGitLab {
+    owner = "pgjones";
```

Comment:
Quart is now a Pallets project, hence the repo moved to https://github.com/pallets/quart

Reply:
Changed source to `github.com/pallets/quart`.
Bumped up version to `0.18.0`.
Changed PR target to `staging`

Thanks :-)